### DISCUSSION TITLE
При смене варианта оплаты "слетает" выбранный ПВЗ

### DISCUSSION THREAD

**Discussion by Xprinter-dv** [cdek-it/wordpress#41] [Q&A]
Неприятный баг, при оформлении сначала идёт выбор варианта доставки и ниже вариант оплаты. Покупатель выбирает СДЭК, выбирает ПВЗ и ниже меняет вариант оплаты, при этом выбранный ПВЗ очищается, нужно снова выбирать.

**Comment by vermorag** [comment]
Добрый день. Информация о ПВЗ не сохраняется в сессию клиента. Однако она перезатирается только когда меняется список доступных вариантов доставки.
Можете подробнее описать всю последовательность действий? Локально воспроизвести не получилось.

**Comment by Xprinter-dv** [comment]
При смене варианта оплаты, обновляются методы доставки, т.к. они зависят от варианта оплаты.
Вы можете попробовать последовательность действий сами:
https://xprinter-dv.ru/ добавляем в корзину любой товар, на странице оформления заказа выбираем город появляются методы доставки СДЭК, выбираем до ПВЗ и выбираем сам ПВЗ. Ниже меняем вариант оплаты, методы доставки обновляются и выбранный ПВЗ сбрасывается.
